packages/blocks/anodot/src/lib/api-filters/get-x-recommendation-property.ts

```diff
@@ -15,58 +15,114 @@ export function getPredefinedRecommendationsDropdownProperty() {
 }
 
 const recommendationTypes = [
-  /* {
+  {
     label: 'AMI Orphaned Snapshot',
-    type_id: [],
-  },*/
+    filters: { type_id: ['ami-orphaned-snapshot'] },
```

Comment:
I really hate this, is there any use case where we can't just do 
`value: 'ami-orphaned-snapshot'` as the dropdown value? (will require migration, but much simpler for the user when changing to dynamic)

Reply:
Then let's simplify and change it to just
{ label: 'AMI Orphaned Snapshot', value: 'ami-orphaned-snapshot' }